src/index.html

```diff
@@ -22,6 +22,362 @@
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                data-qa="nav-hover"
+                class="nav__link qa_hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <div class="cards-container">
+        <div
+          data-qa="card"
+          class="card"
+        >
+          <img
+            src="../src/images/imac.jpeg"
+            alt="Imac"
+            class="card__img"
+          />
+          <h2 class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__rate rate">
+            <div class="rate__stars">
+              <div class="rate__stars-star"></div>
+              <div class="rate__stars-star"></div>
+              <div class="rate__stars-star"></div>
+              <div class="rate__stars-star"></div>
+              <div class="rate__stars-star"></div>
+            </div>
+            <p class="rate__text">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <h2 class="card__price-sum">$2,199</h2>
+          </div>
+          <div class="card__button">
+            <button
+              href="#"
+              class="card__button-buy"
+              data-qa="card-hover"
+            >
+              Buy
+            </button>
+          </div>
+        </div>
+        <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
        </div>
        
        <div class="card">
```